python/hail/docs/getting_started_developing.rst

```diff
@@ -0,0 +1,66 @@
+.. _sec-getting_started_developing:
+
+==========================
+Getting Started Developing
+==========================
+
+You'll need:
+
+- The `Java 8 JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_.
+- `Spark 2.0.2 <http://spark.apache.org/downloads.html>`_
+- `Anaconda <https://www.continuum.io/downloads>`_.
+
+-------------------
+Building a Hail JAR
+-------------------
+
+See :ref:`building-hail-from-source`.
+
+-----------------
+Building the Docs
+-----------------
+
+Hail uses conda environments to manage python dependencies. First, create a
+conda environment for hail:
+
+.. code-block:: bash
+
+    conda env create hail -f ./python/hail/environment.yml
+
+And run the ``createDocs`` gradle task in the environment:
+
+.. code-block:: bash
+
+    (source activate hail && gradle createDocs) ; source deactivate hail
```

Comment:
OK, in that case I'm good with this, although I think we should just document the activate/deactivate, and then write the example gradle commands assuming the Hail environment, e.g. `hail $ ./gradlew createDocs`.